devices.js

```diff
@@ -138,6 +138,42 @@ const execute = (device, actions, callback, delay) => {
 };
 
 const devices = [
+    // Hampton Bay Fan Controller
+    {
+        zigbeeModel: ['HDC52EastwindFan'],
+        model: '99432',
+        vendor: 'Hampton Bay',
+        description: 'Universal wink enabled white ceiling fan premier remote control',
+        supports: 'on/off',
+        fromZigbee: [
+            fz.generic_state, fz.light_brightness, fz.ignore_fan_change, fz.generic_fan_mode, fz.ignore_onoff_change,
+            fz.ignore_level_report,
+        ],
+        toZigbee: [tz.on_off, tz.light_onoff_brightness, tz.light_brightness, tz.fan_mode],
```

Comment:
I think these should be removed: `tz.light_onoff_brightness, tz.light_brightness`.

Reply:
I am not sure if tz.light_onoff_brightness is needed, but tz.light_brightness should remain as the fan does have a light attached and the brightness command does function.

Reply:
Ok, I wasn't aware of the light, about the on/off converter, is it used to turn on/off the fan or the light? (meaning that the `fan_mode` converter is only used to control the fan).

Reply:
I believe that tz.on_off controls just the light.  Assuming when I pass "state: off" to qitt that is sends a tz.on_off command.  When I have controlled the fan it has always been through fan_mode.  There is a button on the remote that comes with the controller itself, that is a "all" power button, which turns off the light and the fan all at once.  I have never used it, but it is there.